manifests/a/app/vleer/1.0.0-beta.3/app.vleer.installer.yaml

```
@@ -0,0 +1,14 @@
+# Created using wingetcreate 1.6.1.0
+# yaml-language-server: $schema=https://aka.ms/winget-manifest.installer.1.6.0.schema.json
+
+PackageIdentifier: app.vleer
+PackageVersion: 1.0.0-beta.3
+InstallerLocale: en-US
+InstallerType: wix
+ProductCode: '{8905B882-D647-4593-946E-5537499CCFBA}'
+Installers:

```

```suggestion
AppsAndFeaturesEntries:
  DisplayVersion 1.0.0
Installers:
```

Oh, then a `DisplayVersion` is needed. This way, the package manager will show `1.0.0-beta.3` when listing and describing installed packages, but will know to match with `1.0.0` in the Registry.